### ISSUE TITLE
Add power normalization

### ISSUE THREAD

**Issue by floscha** [floscha/tabular-dataset#61]
Add a new `power` method to the `numeric.normalize` transformation, based on scikit-learn's [PowerTransformer](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.PowerTransformer.html).

Also, rename the current `power` transformation to `exp` to avoid ambiguity.
